### ISSUE TITLE
prebuilds stopped working

### ISSUE THREAD

**Issue by konne** [gitpod-io/gitpod#8118]
### Bug description

I only get prebuilds in my team if I manually trigger them:

![image](https://user-images.githubusercontent.com/1100969/153194302-2d80b3a0-a976-4f3c-ba5e-8b7e2857a2c4.png)

until 16 days ago (15 from yesterday) it stopped to have automatic prebuilds:

![image](https://user-images.githubusercontent.com/1100969/153194404-3992e294-1ccd-4a81-a55a-9898ad3fe9f1.png)

incremental preBuilds are turned off:

![image](https://user-images.githubusercontent.com/1100969/153195150-acd7c9c9-3957-445c-bbf5-4aa0c512070b.png)

my gitpod yaml:
```
github:
  prebuilds:
    # enable for the default branch (defaults to true)
    master: true
    # enable for all branches in this repo (defaults to false)
    branches: true
    # enable for pull requests coming from this repo (defaults to true)
    pullRequests: true
    # enable for pull requests coming from forks (defaults to false)
    pullRequestsFromForks: false
    # add a check to pull requests (defaults to true)
    addCheck: false
    # add a "Review in Gitpod" button as a comment to pull requests (defaults to false)
    addComment: false
    # add a "Review in Gitpod" button to the pull request's description (defaults to false)
    addBadge: false
```

### Steps to reproduce

look into my tenant, you will find it via my github username (konne)

**Comment by jldec** [comment]
Similar to issue #7648, internal [tracing](https://ui.honeycomb.io/gitpod/datasets/gitpod-production/result/jN9zAYcSZyT) shows auth failures in fetchConfig starting January 24.

One hypothesis is that this Gitpod project belongs to a team which has an owner who does not have GitHub access to the repo in question. @konne could you check whether this may be true? The owners would appear in the members list at `https://gitpod.io/t/<team>/members`.

**Comment by konne** [comment]
does all owners have to have github / gitpod access?
Do you choose the first one of the list?

![image](https://user-images.githubusercontent.com/1100969/153249081-c75ef149-ef6c-43f3-bcd6-3890361fb639.png)

**Comment by jldec** [comment]
We pick the "first" owner returned from the DB, but I'm not sure what the order is unfortunately (it's a bit tricky to ascertain). If you could test whether this owner auth hypothesis from your side that would be great. I'll also try to follow up from our side to improve our logging and error reporting or find another way to get past this.

**Comment by jldec** [comment]
@konne Please feel free to reach out via email at contact@gitpod.io if you prefer to discuss more privately.

**Comment by konne** [comment]
@jldec 
the first owner approach is not really a good one and I assume that is already the issue.
Can you please extend the rights and have an admin and owner.
Owner is the person who need to have the github connection (please also check and show if not okay)
and the admin can also administer the team, but is not the owner.

**Comment by konne** [comment]
@jldec I can confirm. Our admin now connected the github account to gitpod and now its instantly working.

**Comment by jldec** [comment]
Thank you for following up on that @konne - that is very helpful!

We will look into your suggestion to support both **admins** and **owners** in order to prevent this problem from recurring.

**Comment by jldec** [comment]
Closing this as resolved, tracking root cause in #8128.
